src/client/object/appendActions.js

```diff
@@ -5,19 +5,20 @@ import { getVotePercent } from '../reducers';
 
 export const APPEND_WAIVIO_OBJECT = createAsyncActionType('@append/APPEND_WAIVIO_OBJECT');
 
-export const appendObject = (postData, { follow } = { follow: false }) => (dispatch, getState) => {
+export const appendObject = (postData, { follow, notLike } = {}) => (dispatch, getState) => {
```

Comment:
it looks a little weird. Use please `like` or `isLike` name instead of `notLike`